packages/azure-http-specs/specs/azure/resource-manager/resources/mockapi.ts

```diff
@@ -248,9 +254,12 @@ function getUri(resourceUri: string, extensionName: string, requestMethod: strin
 // extension tracked resource
 Scenarios.Azure_ResourceManager_Resources_ExtensionsResources_get = withServiceKeys([
   "ResourceGroup",
+  "SplitResourceGroup",
   "Subscription",
+  "SplitSubscription",
   "Tenant",
   "Resource",
+  "SplitResource",
```

Comment:
With this change, user still have to pass both `SplitXX` and `XX` tests. This is the same as `passOnSuccess`.

Reply:
Rollback to [bd42d17](https://github.com/Azure/typespec-azure/pull/2203/commits/bd42d17208014816dc40104885eff013266f1177)